control/statefbk.py

```diff
@@ -677,9 +677,9 @@ def create_statefbk_iosystem(
     -------
     ctrl : NonlinearIOSystem
         Input/output system representing the controller.  This system
-        takes as inputs the desired state `xd`, the desired input
-        `ud`, and either the system state `x` or the estimated state
-        `xhat`.  It outputs the controller action `u` according to the
+        takes as inputs the desired state :math:`x_d`, the desired input
+        :math:`u_d`, and either the system state :math:`x` or the estimated state
+        :math:`\hat{x}`.  It outputs the controller action :math:`u` according to the
         formula :math:`u = u_d - K(x - x_d)`.  If the keyword
```

Comment:
I would leave this unchanged and convert the formula on line 683 to remove the :math: directive (so `u = ud - K(x - xd)`).